analyzer/quality-analyzer/src/main/java/eu/fasten/analyzer/qualityanalyzer/data/QAConstants.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package eu.fasten.analyzer.qualityanalyzer.data;
+
+public class QAConstants {
+
+    public static final String QA_VERSION_NUMBER = "0.0.1";
+    public static final String QA_PLUGIN_NAME = "Quality Analyzer Plugin";
+
+    //needs to be integrated with core.data.Constants
+    public static final String MVN_FORGE = "mvn";
```

Comment:
I think that these 3 forges should be placed in that `Constants` class we have in the `core/data` because they will be used by multiple plugins. And Maven forge is already there.
Other constants can be here as they are.

Reply:
I agree, the comment says it all (`//needs to be integrated with core.data.Constants`)

Reply:
OK, done